docs/code-of-conduct.md

```diff
@@ -137,7 +137,7 @@ If you see a violation of our code of conduct, [please report it to our
 CLAM Code of Conduct Stewards using this form.](https://forms.gle/7MR7tCpDuprtStWP9)
```

Comment:
Is this indeed a form that ends up with the Executive Council members? Has this been tested? I haven't seen anything